lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,41 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ *
+ * @return string
+ *
+ * @throws Twig_Error_Runtime When an type is not a string or when an invalid trimming type is used (not 'left', 'right' or 'both')
```

Comment:
```diff
-When an type
+When trimming type
```

IMO a shorter sentence like _"When an invalid trimming type is used"_ would be enough, but I'm fine with this one.